### DISCUSSION TITLE
Query regarding SWE assimilation in LIS

### DISCUSSION THREAD

**Discussion by jayaisro** [NASA-LIS/LISF#1652] [Q&A]
Greetings LIS community!

I wanted to assimilate the AMSR-E snow water equivalent (SWE) product from 2005-2008 over greenland. I came accross the LIS test cases for soil moisture which indicates CDF generation (model and observation) and then assimilate with EnKF over the concered region.

I wanted to know if the process is the same for SWE assimilation or if there are test cases I could follow so that I can implement assimilation for my study.

Any help in this is deeply appreciated! 
Thankyou

**Comment by LIS-navari** [comment]
Hi,

The short answer is no.
CDF matching is not required for snow DA.
Currently, we do not have a glacier/snow-ice model capable of running over the GrIS in LIS. However, you are welcome to implement one. We have already integrated Crocus into LIS for seasonal snow, and it works well. I am currently working on a modified version of Crocus to run over the GrIS, specifically for assimilating the ICESat-2 product.
Once you implement the snow/ice model, you will also need to develop interfaces to connect the model to the DA subsystem in LIS.

Best,
Mahdi
